### PR TITLE
ROX-16743: Notifier Side pkg Refactor

### PR DESCRIPTION
**Pull request by keyallis**
## Description

Title.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added

## Testing Performed

Manually verified through UI.

### CODE REVIEW

File: pkg/notifier/notifier_set.go [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
 )
 
-var (
-	integrationSAC = sac.ForResource(resources.Integration)
-)
-
-// NotifierSet is a set that coordinates present policies and notifiers.
-type NotifierSet interface {
+// Set is a set that coordinates present policies and notifiers.
+type Set interface {
```
Comment (clickboo) [inline pkg/notifier/notifier_set.go:13]: Why not leave the name as NotifierSet? Set is too generic.

Comment (clickboo) [reply pkg/notifier/notifier_set.go:13]: Drat, this is the same roxcet naming problem (notifier.NotifierSet doesn't fly) right?

**Review by clickboo — COMMENTED** [review]
How about adding a directory named `processor` under pkg/notifier. That would enable you to keep the name NotifierSet.

File: pkg/notifier/notifier_set.go (hunk 1 above)
Comment (keyallis) [reply pkg/notifier/notifier_set.go:13]: Style checks didn't like it starting with Notifier when the package is named notifier

Comment (clickboo) [reply pkg/notifier/notifier_set.go:13]: Resolved in slack conversation - the naming convention Notifier.Set should work

**Review by clickboo — APPROVED** [review]

**Pull request closed without merging**